contrib/win32/win32compat/shell-host.c

```diff
@@ -1321,16 +1323,16 @@ start_withno_pty(wchar_t *command)
 		process_input = TRUE;
 	else {
 		command_len = wcsnlen_s(command, MAX_CMD_LEN);
-		if ((command_len >= 3 && wcsncmp(command, L"cmd", 4) == 0) ||
-		    (command_len >= 7 && wcsncmp(command, L"cmd.exe", 8) == 0) ||
-		    (command_len >= 4 && wcsncmp(command, L"cmd ", 4) == 0) ||
-		    (command_len >= 8 && wcsncmp(command, L"cmd.exe ", 8) == 0))
+		if ((command_len >= 3 && _wcsnicmp(command, L"cmd", 4) == 0) ||
+		    (command_len >= 7 && _wcsnicmp(command, L"cmd.exe", 8) == 0) ||
+		    (command_len >= 4 && _wcsnicmp(command, L"cmd ", 4) == 0) ||
+		    (command_len >= 8 && _wcsnicmp(command, L"cmd.exe ", 8) == 0))
 			process_input = TRUE;
 	}
 
 	/* Try launching command as is first */
-	if (command) {
-		ret = CreateProcessW(NULL, command, NULL, NULL, TRUE, 0, NULL, NULL, &si, &pi);
+	if (command) {		
```

Comment:
remove the extra spaces like this (after { ).. 
There are multiple places like this.